src/storage/storage_info.cpp

```diff
@@ -2,7 +2,7 @@
 
 namespace duckdb {
 
-const uint64_t VERSION_NUMBER = 41;
+const uint64_t VERSION_NUMBER = 42;
```

Comment:
Is the storage version increase necessary?

Reply:
We can regenerate the database without bumping the storage version, no?

Reply:
Sure, but users that have JSON in their db will crash upon opening it when they update.

Reply:
Hm, that also means that it is no longer possible to store JSON - at least until we fix #4578, right?

Reply:
I've added a test for this and it seems to work properly